[Zeebe] Removed resource from Deployment:CREATED event (#2344)
camunda/camunda-platform-docs: docs/guides/update-guide/820-to-830.md

---
id: 820-to-830
title: Update 8.2 to 8.3
description: "Review which adjustments must be made to migrate from Camunda Platform 8.2.x to Camunda Platform 8.3.0."
---

<span class="badge badge--primary">Intermediate</span>

The following sections explain which adjustments must be made to migrate from Camunda Platform 8.2.x to 8.3.0 for each component.

## Zeebe

> ### Resource not more available in Deployment:CREATED event

Reviewer: ```suggestion
### Resources no longer available in the Deployment:CREATED event
```

> Resource into **Deployment:CREATED** event are no longer available. Resource are available in **Process:CREATED** events for BPMN models or in **DecisionRequirements:CREATED** events for DMN models

Reviewer: ```suggestion
Resources in the **Deployment:CREATED** event are no longer available. Custom Exporters using the resources from this event must be modified to get them from the **Process:CREATED** event for BPMN models and the **DecisionRequirements:CREATED** event for DMN models.
```
